editor/plugins/gizmos/camera_3d_gizmo_plugin.cpp

```diff
@@ -212,6 +212,28 @@ void Camera3DGizmoPlugin::redraw(EditorNode3DGizmo *p_gizmo) {
 			ADD_TRIANGLE(tup, side + up, nside + up);
 		} break;
 
+		case Camera3D::PROJECTION_OBLIQUE: {
+			// Mirrors the PROJECTION_PERSPECTIVE because they are fundamentally the same.
+			float fov = camera->get_fov() / 2.0;
+
```

Comment:
If this is the same code as `PROJECTION_PERSPECTIVE` then the code should be shared using multiple case statements.
```
case Camera3D::PROJECTION_PERSPECTIVE:
case Camera3D::PROJECTION_OBLIQUE:
{
```

Reply:
this will be updated in the next push